guides/common/modules/proc_salt-enabling-salt-grains-upload.adoc

```diff
@@ -0,0 +1,27 @@
+[id="salt_guide_enabling_salt_grains_upload_{context}"]
+= Enabling Salt Grains Upload
+
+Managed hosts running the Salt Minion client software can upload Salt Grains to {ProjectServer} or {SmartProxyServer}.
+Salt Grains are analogous to Puppet facts.
```

Comment:
Can we always be so sure that a Salt user understands Puppet terminology? I would suggest that spelling out what we mean here would be safer.

Reply:
Fixed.